test/expected_results/test_class_shapes.ttl

```diff
@@ -0,0 +1,71 @@
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix sh:  <http://www.w3.org/ns/shacl#> .
+@prefix td:  <http://test.data#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+td:TestClassAShape
+    a               sh:NodeShape ;
+    sh:property     [ sh:class td:TestClassD ;
```

Comment:
I'd suggest property shapes get their own IRI. I've often needed to refer to them directly.